DataFormats/L1TrackTrigger/interface/TTTrack.h

```diff
@@ -63,7 +63,7 @@ class TTTrack : public TTTrack_TrackWord {
           double aChi2,
           double trkMVA1,
           double trkMVA2,
-          double trkMVA3,
+          double trkMVA3_,
```

Comment:
@mdhildreth why did trkMVA3 change to trkMVA3_ in the two constructors? This seems odd to me?

Reply:
oh, that's a typo.  Shouldn't be...  It doesn't really matter, but...